packages/wallets/src/components/WalletsResetMT5Password/WalletsResetMT5Password.tsx

```diff
@@ -79,51 +81,61 @@ const WalletsResetMT5Password = ({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [platform, title, actionParams, isChangeInvestorPasswordSuccess, isChangeInvestorPasswordError]);
 
-    const renderFooter = () => {
-        return isMobile ? (
-            <WalletButtonGroup isFullWidth>
-                <WalletButton onClick={() => hide()} size='lg' variant='outlined'>
-                    <Trans defaults='Cancel' />
-                </WalletButton>
-                <WalletButton
-                    disabled={!validPassword(password)}
-                    isLoading={isChangeInvestorPasswordLoading || isChangePasswordLoading}
-                    onClick={handleSubmit}
-                    size='lg'
-                    variant='contained'
-                >
-                    <Trans defaults='Create' />
-                </WalletButton>
-            </WalletButtonGroup>
-        ) : null;
-    };
+    const renderFooter = isMobile
+        ? () => {
+              return (
+                  <WalletButtonGroup isFullWidth>
+                      <WalletButton onClick={() => hide()} size='lg' variant='outlined'>
+                          <Trans defaults='Cancel' />
+                      </WalletButton>
+                      <WalletButton
+                          disabled={!isValidPassword}
+                          isLoading={isChangeInvestorPasswordLoading || isChangePasswordLoading}
+                          onClick={handleSubmit}
+                          size='lg'
+                          variant='contained'
+                      >
+                          <Trans defaults='Create' />
+                      </WalletButton>
+                  </WalletButtonGroup>
+              );
+          }
+        : undefined;
```

Comment:
Fixed an issue where the footer is visible:
<img width="659" alt="Screenshot 2024-04-04 at 1 57 27 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/e1e4007d-8666-462b-b0c8-c8ae235ae858">